identity-service/src/routes/wormhole.js

```diff
@@ -0,0 +1,103 @@
+const { sendResponse, successResponse, errorResponseBadRequest, errorResponseServerError } = require('../apiHelpers')
+const ethTxRelay = require('../relay/ethTxRelay')
+const crypto = require('crypto')
+const { getFeePayer } = require('../solana-client')
+
+const { NodeHttpTransport } = require('@improbable-eng/grpc-web-node-http-transport')
+
+const checkContract = (method) => {
+  return method.contractAddress && method.encodedABI && method.gasLimit
+}
+
+const getTxProps = (senderAddress, method) => {
+  const sha = crypto.createHash('sha256').update(JSON.stringify({ senderAddress, ...method })).digest('hex')
+  return {
+    sha,
+    txProps: {
+      contractAddress: method.contractAddress,
+      encodedABI: method.encodedABI,
+      senderAddress: senderAddress,
+      gasLimit: method.gasLimit || null
+    }
+  }
+}
+const feePayerAccount = getFeePayer()
+
+module.exports = function (app) {
+  app.post('/wormhole_relay', async (req, res, next) => {
+    const audiusLibs = req.app.get('audiusLibs')
```

Comment:
There is already a separate method that does the same thing in libs as here. ref: https://github.com/AudiusProject/audius-protocol/blob/cc902d8127e753d0b7ae4414305f7695c94f1076/libs/src/api/account.js#L388-L436
 
I think it might be too messy to make it a single method that is generic to both proxy and sending off the transactions locally because it is multiple steps. There are two eth transactions - the second relying on the first and we need an extra step on the proxy side to estimate gas as well. 

Note that the entire sol tx side of things uses the same libs method: `wormholeClient.attestAndCompleteTransferEthToSol`